Assignment2/JustAntoRS-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,50 @@
+{
+    "@context":{
+        "xsd":"http://www.w3.org/2001/XMLSchema#",
+        "prop": "http://www.examples.org/properties"
```

Comment:
Add / or # at the end of URI definition

Reply:
Thanks for spotting out the typo, fixed it :)